gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/avro/AvroSchemaManager.java

```diff
@@ -42,6 +45,9 @@
 import org.apache.gobblin.hive.avro.HiveAvroSerDeManager;
 import org.apache.gobblin.util.AvroUtils;
 import org.apache.gobblin.util.HadoopUtils;
+import org.apache.hadoop.hive.serde2.avro.TypeInfoToSchema;
```

Comment:
These should be grouped with the org imports above. We keep org.apache.gobblin separate from org.